templates/topic/index.json-ld

```diff
@@ -299,13 +299,13 @@
 			"example": {
 				"en": [
 					{
-						"href": "demoted/theme-topic-en.html",
+						"href": "deprecated/theme-topic-en.html",
 						"text": "Theme - Topic page version 2 (beta)"
```

Comment:
We are missing a change set to hold the state of those iteration.

For now, let update the title and clearly indicate they are deprecated.

```suggestion
						"href": "deprecated/theme-topic-en.html",
						"text": "Deprecated as GCWeb v16.0.0 - Theme - Topic page version 2 (beta)"
```